apps/analysis/translation.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+from apps.teams.utils import current_team
+
+from .const import LANGUAGE_CHOICES
+
+TRANSLATION_PROMPT_TEMPLATE = """Translate these chat messages to {target_lang_name}.
+Return a JSON array where each object has a "translation" field.
```

Comment:
This needs to also include the ID now since you're using it in the output
```suggestion
Return a JSON array where each object has the following fields:
- id: The ID of the message (must match the ID of of the input message)
- translation: The translated text
```